Shoko.Plugin.Abstractions/Attributes/RenamerAttribute.cs

```diff
@@ -8,6 +8,7 @@ public class RenamerAttribute : Attribute
         public RenamerAttribute(string renamerId)
         {
             RenamerId = renamerId;
+            _desc = renamerId;
```

Comment:
Looking at how it's used, then it should probably be set to `null` and not the renamer id if not explicitly set by a renamer, or we can change up the `Description` to not use a private field and instead just be set to the renamer id initially like you did here.

Reply:
I just set it to renamerId because of how the getter for it was coded, can set to empty string or make nullable.

Reply:
Please mark it as nullable then.

Reply:
I'd suggest doing this instead: https://github.com/ShokoAnime/ShokoServer/pull/1153/files#r1685748964